src/content/whats-new/2022/01/whats-new-errors-inbox-browser-mobile-1-20.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Errors Inbox: Now in Browser and Mobile' 
+summary: 'Triage Browser and Mobile errors without setting up a workload' 
+releaseDate: '2022-01-20' 
+getStartedLink: 'https://docs.newrelic.com/docs/errors-inbox/errors-inbox/'
+---
+
+New Relic Errors Inbox is an error tracking solution that provides a single place to view, triage, and resolve errors across the full application stack.
+Now we have added Errors Inbox to the Browser and Mobile UIs, which makes triaging Browser and Mobile errors a breeze without the need to set up a workload.
+
+Watch an extended demo in our recent Nerdlog episode below to learn more about how errors inbox can help you track your errors intelligently:
+<Video id="M52jeJsnOfU" type="youtube" />
```

Comment:
whats new posts should use `<iframe>` and not Video component. see other whats new posts as example